src/plus/launchpad/launchpadProvider.ts

```diff
@@ -1033,7 +1034,7 @@ export function getPullRequestBranchDeepLink(
 			DeepLinkType.Branch
 		}/${encodeURIComponent(headRefBranchName)}?url=${encodeURIComponent(ensureRemoteUrl(remoteUrl))}${
 			action != null ? `&action=${action}` : ''
-		}`,
+		}${prId != null ? `&prId=${prId}` : ''}`,
```

Comment:
I'm not sure if a PR id can ever have characters that would need to be encoded, but just in case, I suggest using `encodeURIComponent` on it.

Reply:
I suggest using native URLSearchParams class as more readable and correct way to add search parameters to the url string

Reply:
It looks like the link generation should be moved on deeplinkink service side. I would also make the `UriTypes` enum instead of using satisfies

Reply:
Both of those suggestions sound good, thanks!